src/components/Home/Row.js

```diff
@@ -1,39 +1,47 @@
 import React, { useState } from "react";
-import { makeStyles } from "@material-ui/core/styles";
 import Card from "@material-ui/core/Card";
 import CardActionArea from "@material-ui/core/CardActionArea";
 import CardMedia from "@material-ui/core/CardMedia";
-
 import { HOME_CARD_IMG } from "../../constant/properties";
 import "./Home.css";
+import { useTranslation } from "react-i18next";
 
-const useStyles = makeStyles({
-  root: {
-    maxWidth: 455,
-    maxHeight: 300,
-  },
-});
+function CategoryType({ category, type }) {
+  const [t] = useTranslation("common");
+  const { tagline, image } = category;
+  return (
+    <>
+      <Card className="card">
+        <CardActionArea>
+          <CardMedia
+            component="img"
+            alt={tagline}
+            image={image}
+            title={t(`header.categorytype.${type}.${tagline}`)}
+          />
+          <div className="row__category__tag">
+            <span className="row__category__tag__span">
+              {t(`header.categorytype.${type}.${tagline}`)}
+            </span>
+          </div>
+        </CardActionArea>
+      </Card>
+    </>
+  );
+}
 
 function Row({ type }) {
-  const classes = useStyles();
+  const [t] = useTranslation("common");
 
   return (
-    <div className="row">
-      {HOME_CARD_IMG[0][type].map((category) => (
-        <Card className={classes.root}>
-          <CardActionArea>
-            <CardMedia
-              component="img"
-              alt={category.tagline}
-              height="340"
-              image={category.image}
-              title={category.tagline}
-            />
-            <div className="row__category__tag">{category.tagline}</div>
-          </CardActionArea>
-        </Card>
-      ))}
-    </div>
+    <>
+      <h1 className="row__type">{t(`header.categorytype.${type}.title`)}</h1>
+      <div className="row">
+        {HOME_CARD_IMG[0][type].map((category) => (
```

Comment:
This `HOME_CARD_IMG[0]`, can we give a better name to this?

Reply:
yes, okay